checkout.js

```diff
@@ -0,0 +1,10 @@
+const form = document.querySelector('form');
+
+form.addEventListener('submit', function(e) {
+  e.preventDefault();
+  alert("Thank you. The form information has been received");
+  var divs = document.querySelectorAll('input');
+  for (let i = 0; i < divs.length; ++i) {
```

Comment:
There is a delicate difference between ++i and i++. For now, make sure you're using i++ or you could get weird results